build/xtask/src/dist.rs

```diff
@@ -84,7 +84,7 @@ pub fn package(
         }
     }
     for (name, range) in &memories {
-        println!("{:<5} = 0x{:0>8x}..0x{:0>8x}", name, range.start, range.end);
+        println!("{:<5} = {:0>#8x}..{:0>#8x}", name, range.start, range.end);
```

Comment:
I don't think this one works:

```rust
    let x = 12345;
    println!("{:0>#8x}", x);
```

prints

```
000x3039
```

Reply:
Looks like `{:0>#010x}` works though

Reply:
Now that I think about it, I don't think the `0>` is even necessary, since we're using `0` padding, which necessarily goes onto the left; it seems like `{:#010x}` should work fine.